src/scm/bitbucket.js

```diff
@@ -0,0 +1,4 @@
+"use strict";
+
+const Bitbucket = class Bitbucket {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/9b7bf2b597e67365f1e9002c784237b58657e4cc/src/scm/bitbucket.js#L3"></span>‘Bitbucket’ is assigned a value but never used.</td>
    </tr>
  </tbody>
</table>